packages/eslint-config-webservices/rules/react.js

```diff
@@ -31,5 +31,9 @@ module.exports = {
     'react/forbid-foreign-prop-types': 'error',
     'react/no-direct-mutation-state': 'error',
     'react/no-access-state-in-setstate': 'error',
+    'react/sort-prop-types': ['warn', {
```

Comment:
Do we need `warn` or `error` level for this rule? In other words, do we want the violation of this rule to fail a CI build or just want to get comments in a PR?
It's ok to add the new rules that produce errors if it's our desired final state. For the projects that are not compliant with it, we always can make the local override for the transition period.

Reply:
For me `error` level is more suitable, but I was afraid for integration in other projects.

> For the projects that are not compliant with it, we always can make the local override for the transition period.

If so, let's change to `error` level. What do you think?

Reply:
I vote for `error`.

Reply:
Done